ew/utils/loop.py

```diff
@@ -858,6 +859,87 @@ async def enemy_action_tick_loop(id_server):
+                        else:
+                            slimeloss = min(250,slime)
+                            injury_text = ["{name} suddenly faceplants onto the floor from standing leaving a nasty bruise. **They lose {slime} slime!**".format(name=player.display_name, slime = slimeloss)]
+                    
+                    
+                    elif inebriation<=severityLimit.get("wasted"):
+                        if savingthrow > 15:
+                            injury_text = ["{name} attempts to walk, stumbles slightly but manages to correct themselves. Good job!".format(name=player.display_name,slime=slimeloss),"{name} eructates.".format(name=player.display_name)]
+                        else:
+                            slimeloss = slime * 0.05 # 5% slimeloss
+                            injury_text = ["{name} attempts to walk, accidentally standing on the side of their foot. Their ankle is almost instantly sprained and in their pained hopping they manage to topple over a nearby railing falling backfirst onto the pavement. **They lose {slime} slime!**".format(name=player.display_name,slime=slimeloss)]
+                    
+                    
+                    elif inebriation<=severityLimit.get("wrecked"):
+                        if savingthrow > 18:
+                            injury_text = ["{name} drunkenly punches a nearby window but the glass doesn't break. Phew!".format(name=player.display_name,slime=slimeloss)]
+                        else:
+                            slimeloss = slime * 0.1 # 10% slimeloss
+                            injury_text = ["{name} drunkenly punches a nearby window, the glass shatters and their hand is cut to ribbons by the resultant shards. **They lose {slime} slime!**".format(name=player.display_name,slime=slimeloss)]
+                    
+                    
+                    else:
+                        if savingthrow == 20:
+                            injury_text = ["{name} mindlessly stumbles out into a nearby road. An oncoming car bleats and {name} hears it! Only just scrambling away from the oncoming car. Nice save!".format(name=player.display_name)]
+                        else:
+                            slimeloss = slime * 0.2 # 20% slimeloss
+                            injury_text = ["{name} mindlessly stumbles out into a nearby road. An oncoming car bleats but they are too inebriated to hear it, their hip is clipped by the cars headlight. **SMASH!!** They are sent spiralling into the air! Spinning at least 3 full revolutions before falling back onto the pavement with a healthy *crunch*. They spew out the contents of their stomach as they stumble back to their feet. A symphony of slime, bile and blood coats the asphalt. **They lose {slime} slime!**".format(name=player.display_name, slime=slimeloss)]
```

Comment:
comma between bleats and but. "and" before "their hip". "car's" instead of "cars". Replace "air!" with "air,"
Oxford comma between "bile" and "and blood".